### DISCUSSION TITLE
The "Status Monitor" misrepresents the oxygen ratio in the two "hulls" if were "linked".

### DISCUSSION THREAD

**Discussion by MaaiG** [FakeFishGames/Barotrauma#7337] [Unstable feedback]
How to do it:
1. You have two hulls and there’s a fire door between them.
2. There’s a vent in the right hull.
3. In order to display these hulls on a minicard as one space, you establish a link between them (Spacebar).
4. In the left hull, a fire breaks out and the fire door closes to separate the right hull. 
5. There’s no ventilation in the left hull and the door’s closed.

The problem with "Status monitor" - incorrectly displays ratio of percentage of oxygen in connected spaces. If in reality in the left hull = 0% oxygen and in the right hull = 100% oxygen, but the "Status Monitor" shows: Left Hull=33% and Right Hull=67%

![Image 013](https://user-images.githubusercontent.com/93348499/142042656-1ac6efb8-5b35-4520-bfb9-b0617a851dfb.png)
!

**Comment by ghost** [comment]
my guy this is the wrong category, check out the bugs section instead of the suggestions section. Also if you mean gap, those function to let things pass between hulls.

**Reply by StruggleDuck** [reply]
^^

**Comment by MaaiG** [comment]
I have to double-check. Maybe it’s my fault - I have to figure it out.

Upd.
I figured out what it is! Oxygen mechanics works right. Problem with "Status monitor" incorrectly displays ratio of percentage of oxygen in connected spaces. If in reality in the first hull 0% and in the second hull 100% the monitor status shows 33% and 67%.

**Reply by MaaiG** [reply]
The discussion was created in this section by my mistake.
Right section: https://github.com/Regalis11/Barotrauma/issues/7349
> Fixed in [Regalis11/Barotrauma-development@2905397](https://github.com/Regalis11/Barotrauma-development/commit/2905397106e10d9e0c319265d59a8513b7e96568)
